app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/SplashActivity.java

```diff
@@ -51,13 +54,25 @@ protected void onCreate(Bundle savedInstanceState) {
     @Override
     protected void onStart() {
         super.onStart();
-        loadSessionsForCache();
+        // TODO !IMPORTANT Uncomment this line or you will never see the schedule ...
+        // loadSessionsForCache();
+
+        // Starting new Activity normally will not destroy this Activity, so set this up in start/stop cycle
+        Takt.stock(getApplication())
+                .seat(Seat.BOTTOM_RIGHT)
+                .interval(250)
+                .listener(fps -> Log.w("DroidKaigi", "heartbeat() called with: fps = [" + fps + "]"))
```

Comment:
**[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: Using 'Log' instead of 'Timber' [LogNotTimber]